src/components/Header/index.tsx

```diff
@@ -251,7 +251,14 @@ function Header() {
           <HeaderLink data-testid="swap-nav-link" id="swap-nav-link" to="/swap">
             {t('swap')}
           </HeaderLink>
-          <HeaderLink data-testid="pool-nav-link" id="pool-nav-link" to="/pools" disabled={networkWithoutSWPR}>
+          <HeaderLink
+            data-testid="pool-nav-link"
+            id="pool-nav-link"
+            to="/pools"
+            disabled={
+              networkWithoutSWPR && chainId !== ChainId.ARBITRUM_GOERLI
+            } /* // FIXME: fix this once SWPR is on Arb Goerli */
```

Comment:
Shouldn't we update this value inside `SWPR[chainId]`?

Reply:
@luzzif, do we have Swapr on Arbitrum Görli yet?